### PR TITLE
927 add open passport buttons

### LINKED ISSUE #927: Add Open Passport buttons
**User Story:**
As a donor
I want to calculate my Passport score
So that I can know if I am eligible for matching

**Acceptance Criteria**
GIVEN that I am on the Passport Verification page
WHEN I click on the "Open Passport" button
THEN the [Passport app](https://passport.gitcoin.co/#/dashboard) opens in a new tab

**Product & Design Links:**
[Figma](https://www.figma.com/file/lZwHrKYiWJna3AePzyTAiG/Grant-Explorer?node-id=2741%3A76879&t=vLINKnBq7Ra1eJw8-4)

**Tech Details:**

**Open Questions:**

**Notes/Assumptions:**
- This includes writing test coverage for the functionality added in the ticket, if necessary

### PR DESCRIPTION
**Pull request by thelostone-mc**
##### Description

- add open passport / create passport
- added dummy button for recalculate score

##### Refers/Fixes

closes #927

### CODE REVIEW

**Review by vacekj — APPROVED** [review]
LGTM

File: packages/grant-explorer/src/features/api/passport.ts [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+        })
+        .catch((err: Response) => {
+          switch (err.status) {
+            case 400: // unregistered/nonexistent passport address
+              setPassportState(PassportState.INVALID_PASSPORT);
+              console.log(err.json());
+              break;
+            case 401: // invalid API key
+              setPassportState(PassportState.ERROR);
+              console.log(err.json());
```
Comment (chibie) [inline packages/grant-explorer/src/features/api/passport.ts:72]: shouldn't these be `console.error` ?

File: packages/grant-explorer/src/features/round/PassportConnect.tsx [hunk 2]
```diff
@@ -0,0 +1,228 @@
...
+      </Tab.Group>
+    )
+  }
+
+  function PassportButtons(props: { hasPassport: boolean }) {
+    return (
+      <>
+        <div className="flex justify-center">
+          <p>TODO: ADD Passport score and status</p>
+        </div>
```
Comment (chibie) [inline packages/grant-explorer/src/features/round/PassportConnect.tsx:148]: shouldn't this be a comment?

**Review by melissa-neira — COMMENTED** [review]
I can't test because I keep getting this error 😖
screencapture-bafybeia4labunxirvzzucoc2qaogpfw2izfut3x5pah3o3u7i7vxb64pmm-on-fleek-co-2023-01-10-13_14_52

**Comment by thelostone-mc** [comment]
Closing in favour of https://github.com/gitcoinco/grants-round/pull/958/

**Pull request closed without merging**